operator/roles/ansible-service-broker/defaults/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+broker_name: ansible-service-broker
+broker_namespace: openshift-ansible-service-broker
+
+# 3.11 compatible names
+broker_deployment_name: asb
+broker_service_name: asb
+broker_route_name: asb-1338
+dashboard_redirector_route_name: dr-1337
+
+broker_image: docker.io/ansibleplaybookbundle/origin-ansible-service-broker:v3.11
+
+broker_kind: ClusterServiceBroker
+broker_image_pull_policy: "IfNotPresent"
+broker_probe_initial_delay: 120
+broker_node_selector: ""
+broker_basic_auth_enabled: false
+
+_broker_config:
```

Comment:
This is the part that needs to be reworked based on conversation with @eriknelson and @shawn-hurley. Basically we should do what Operators are meant to do and encode the best practices and limit the amount of configuration possible in the broker_config.